distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time.enso

```diff
@@ -583,13 +596,24 @@ type Date_Time
              import Standard.Base.Data.Time.Duration
 
              example_minus = Date_Time.new 2020 - (Period.new years=1) - (Period.new months=9) - (Duration.new hours=5)
+
+       > Example
+         Subtract one quarter from a zoned date time.
+
+             from Standard.Base import Date_Time, Date_Period
+
+             example_plus = Date_Time.new 2020 - Date_Period.Quarter
     - : (Duration | Period) -> Date_Time ! Time_Error
     - self amount =
```

Comment:
as we don't have the default fallback in the case we should add type checking.

Reply:
Done.